doc/source/istanbul.md

```diff
@@ -0,0 +1,32 @@
+
+Istanbul is the country’s economic, cultural, and historic center. In fact, Turkey owes 31% of its total GDP to Istanbul. (according to 2019 reports) Moreover, the city’s economy is bigger than that of 130 countries, 25 of which is located in Europe. When it comes to the culture, much of Turkey’s cultural richness had its roots in Istanbul. It is culturally such an important city that in 2010, it was named as European Capital of Culture. If we need to speak about the history of the city, we can not tell it by words. So, we will not discuss its history besides one really and may be the most important event in the city’s history: The Conquest of Istanbul. This was such a huge event that according to some historians, early modern period of history begins with this event. This is because after the fall of Constantinople, the feudalism system in Europe ended.
+
+The historical importance of Istanbul is mainly due to is geographical location. It is located in north-western Turkey and it is divided into two by the Bosporus, which is the only way to go to the Black Sea from the Mediterranean Sea. So, historically it has been ideally located for trade and defense. Its defensive importance still continues thanks to Montreux Convention Regarding the Regime of the Straits. Thanks to that, the USA cannot send their warships to the Black Sea on which Russia has border. As we have mentioned before, the city connects Asia and Europe, and it is not difficult to say Istanbul is also a cultural bridge between Asia and Europe just like its bridges which connect the two continents.
+
+World Heritage Sites are areas or landmarks with legal protection by an international convention administered by UNESCO and 8 of these sites are located in Istanbul. This must have an effect over tourists because Istanbul is also Turkey’s mostly visited city. In 2018, over 13.4 million foreigner visitors visited the city, and this make the city the world’s fifth-most popular tourist destination.
+
+Istanbul is primarily known for its Byzantine and Ottoman architecture. Even though it is a Turkish city since 1453, the city still contains both Christian and ancient monuments.
+
+The city has 39 districts who have their own councils and mayors, but these districts are administered by the Istanbul Metropolitan Municipality. This system was brought from France by republicans known as young-turks. This system is now applied in all the cities of Turkey.
+
+As Istanbul passed into other hands, it has been a cosmopolitan city throughout the history. It continues to have that characteristic of its, but it has become more homogenized since the Ottoman era. There are more than 2 million of Arabs, from 2 to 4 million of Kurds, 3,000 Greeks, 123,363 Armenians and 25,000 Jews in Istanbul.
```

Comment:
As Istanbul passed into other hands, it has been a cosmopolitan city throughout history. It continues to have that characteristic of its, but it has become more homogenized since the Ottoman era. There are more than 2 million Arabs, from 2 to 4 million Kurds, 3,000 Greeks, 123,363 Armenians, and 25,000 Jews in Istanbul.